### ISSUE TITLE
no "post" or "page" layout

### ISSUE THREAD

**Issue by roninb** [pages-themes/hacker#20]
Most of the pages-themes lack layouts for normal posts (a blog post) or pages (a "contact" or "about" page). 

@benbalter mentioned that there weren't any plans to add them but the maintainer's would be glad to review and possibly merge PRs. Since I was planning on using this theme for my personal site, I'm gonna spend some time trying to get those implemented.

**Comment by ndench** [comment]
Can a new version be released including this layout?

**Comment by micfan** [comment]
```
bundle
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
Fetching gem metadata from https://rubygems.org/................
Fetching gem metadata from https://rubygems.org/..
Resolving dependencies....
Bundler could not find compatible versions for gem "jekyll":
  In snapshot (Gemfile.lock):
    jekyll (= 3.8.5)

  In Gemfile:
    github-pages was resolved to 4, which depends on
      jekyll (= 1.1.2)

    minima (~> 2.0) was resolved to 2.5.0, which depends on
      jekyll (~> 3.5)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```
